code/multiple_cursors.py

```diff
@@ -24,6 +24,9 @@ def multi_cursor_select_fewer_occurrences():
     def multi_cursor_select_more_occurrences():
         """Adds cursor at next occurrence of selection"""
 
+    def multi_cursor_select_skip_occurrences():
```

Comment:
I find this action name a bit confusing.  How about `multi_cursor_skip_occurrence` instead?

Reply:
okey dokey